src/components/PostBar/PostBar.jsx

```diff
@@ -18,6 +20,10 @@ export default function PostBar({
   const navigate = useNavigate();
   const [agoTime, setAgoTime] = useState(timeAgo(data.createdAt));
   const currentBoard = pathname.split('/')[2];
+  const searchBoardId =
+    currentBoard !== 'all'
+      ? currentBoard
+      : BOARD_MENUS.find((menu) => menu.title === data.boardName)?.textId;
```

Comment:
find문에 menu를 객체 구조분해를 이용한다면 코드를 간소화 시킬 수 있습니다 :)

```suggestion
  const searchBoardId =
    currentBoard !== 'all'
      ? currentBoard
      : BOARD_MENUS.find(({ title }) => title === data.boardName)?.textId;
```